### PR TITLE
Update to 1.6

### PR DESCRIPTION
**Pull request by fruitl00p**
I know this seems like a really big PR. But this will update the SDK to 1.6 (including the PR #2 )

The main reason the PR is so big is because of the CRLF -> LF change.

I'm also thinking of fully altering the SDK itself to do the following:
- use short array syntax
- drop `intval` and replace it with `(int)`-typecast
- move everything to a much cleaner namespace (i.e. `opensdks\omnikassa2-sdk` instead of `nl\rabobank\gict\payments_savings\omnikassa_sdk`)
- include the documented implementation of an in memory-`TokenProvider` to have at least a working starting point
- update the docs further / implement some more example code outside of the readme et al

In other words: making either a fully wrapped composer package / straight up creating a full fork of their SDK. (and possibly renaming the package outside of `opensdks` in order to prevent conficts)

Do you have any thoughts?

### CODE REVIEW

**Comment by NielsJanssen** [comment]
Thanks for the contribution! I'll merge and tag as 1.6.0 to match the SDK version.

As for your other suggestions. I'm all for making the SDK more accessible to use and modernising it. My main concern however would be deviating too much from the base SDK, making updating much more of a pain.

Apologies for the slow response by the way, I don't have an active project which needs the SDK right now so it's not been a priority for me. Would you consider becoming a contributor?

**Comment by fruitl00p** [comment]
@NielsJanssen thansk for the merge (and reply) Understandable that this isn't high on the priorities anymore as your not actively implementing the engine.

To answer your questions regarding my proposal: I fully agree that this might impede furture updates. And a possible better approach is to fully wrap the original SDK into a user package that:
- is easily composer-installable
- provides a clear interface and namespace to the outside world
- follows sometype of modern code styling / best practices

This would allow for 'drop-in' replacement of the orignal SDK, but allow the users of the `opensdks\omnikassa2-sdk`-package to just keep working ;) (in best case scenario if everything can be updated under the hood)
